### PR TITLE
feat(app-component): Cleaning up app component and adding a heading file

### PR DESCRIPTION
**Pull request by Alexg1021**
- cleaned out react template code from app.js file
- added a heading and subheading element to the app.js file

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
```
Comment (Alexg1021) [inline src/App.js]: Hey you forgot to change to an arrow function. Do you still want to merge?

**Review by nakiacallaway — CHANGES_REQUESTED** [review]
Change function App to arrow function

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,10 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className='App'>
+      <h1>This is my App Component</h1>
+      <h3>From my ag-1 feature branch!</h3>
```
Comment (Alexg1021) [inline src/App.js:7]: Awesome! like that you reference your branch in the subheading

**Comment by Alexg1021** [comment]
@nakiacallaway made the change! Can you review?

**Review by nakiacallaway — APPROVED** [review]
Looks great.

**Pull request merged**